hack/golangci-hints.yaml

```diff
@@ -67,6 +67,16 @@ issues:
         - gocritic
       text: "ifElseChain: rewrite if-else to switch statement"
 
+    # Only packages listed here opt into the strict "exported symbols must be documented".
+    #
+    # Exclude texts from https://github.com/golangci/golangci-lint/blob/ab3c3cd69e602ff53bb4c3e2c188f0caeb80305d/pkg/config/issues.go#L11-L103
```

Comment:
/lgtm
/hold

drop hold, when see fit.

SGTM, but what would happen if the golintci defaults change, do we need to adapt?

Reply:
We might have to, although it should be rare.

Reply:
/hold cancel